modules/subchannel/src/problems/SubChannel1PhaseProblem.C

```diff
@@ -157,6 +151,10 @@ SubChannel1PhaseProblem::SubChannel1PhaseProblem(const InputParameters & params)
     _duct_heat_flux_soln(nullptr),
     _Tduct_soln(nullptr)
 {
+  if (_pin_mesh_exist && !isParamValid("pin_HTC_closure"))
+    mooseError(name(), ": pin_HTC_closure is required when a pin mesh exists.");
```

Comment:
```suggestion
    paramError("pin_HTC_closure", required when a pin mesh exists.");
```
The problem does not have a helpful name